include/ck_tile/ops/gemm/kernel/gemm_tile_partitioner.hpp

```diff
@@ -59,10 +64,10 @@ struct GemmTile1DPartitioner
 
     CK_TILE_DEVICE auto operator()(index_t blockOffset, index_t NBlockSize)
```

Comment:
The name `NBlockSize` is not reflecting its actual meaning. 
```suggestion
    CK_TILE_DEVICE auto operator()(index_t blockOffset, index_t N)
```